public/app/features/search/components/SearchResultsFilter.tsx

```diff
@@ -42,7 +42,11 @@ export const SearchResultsFilter: FC<Props> = ({
 
   return (
     <div className={styles.wrapper}>
-      {editable && <Checkbox value={allChecked} onChange={onToggleAllChecked} />}
+      {editable && (
+        <div className={styles.checkboxWrapper}>
+          <Checkbox label="Select all" value={allChecked} onChange={onToggleAllChecked} />
```

Comment:
Would `aria-label` be enough here maybe? That label looks a bit out of place there imo.